app.rb

```diff
@@ -50,20 +62,23 @@ def create_rental
     date = gets.chomp
     @rentals << Rental.new(@people[person_index], @books[book_index], date)
     puts 'Rental created successfully'
+    run
   end
 
   def list_books
-    return puts 'No books found!' if @books.empty?
```

Comment:
* I get these bugs when I try to create a person and a rental 
* When I select list all people the name of the person I created doesn't show up also for creating rentals when I select a book from the list by the number I get invalid input. Kindly fix them 
![image](https://user-images.githubusercontent.com/52497246/194896871-2a80892b-2750-472c-a3ac-4222f8e3589a.png)
![image](https://user-images.githubusercontent.com/52497246/194896893-c96663d0-a24d-45be-a405-a0861dc411d0.png)